pkg/mixer/eval.go

```diff
@@ -21,16 +21,37 @@ import (
 )
 
 func evaluatePrometheusAlerts(vm *jsonnet.VM, filename string) (string, error) {
-	snippet := fmt.Sprintf("(import '%s').prometheusAlerts", filename)
+	snippet := fmt.Sprintf(`
+local mixin = (import %q);
+
+if std.objectHas(mixin, "prometheusAlerts")
+then mixin.prometheusAlerts
+else {}
+`, filename)
```

Comment:
```jsonnet
({ prometheusAlerts: {} } + (import '%q')).prometheusAlerts
```

Also your import needs quotes :)

Reply:
> Oneline alternative:

I intentionally did the more verbose style, to make it obvious what I was doing.  Jsonnet it hard enough to read as it is.

> Also your import needs quotes :)

`%q` adds quotes for you, and escapes the string.

Reply:
sgtm

Reply:
actually, I never came across this and others may experience the same. Using `'%s'` would be more obvious, but I don't mind leaving it as is given it's a standard Go feature

Reply:
Using '%s' would break if the filename had a single quote in it.